libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -1072,6 +1086,11 @@ AP_GPS_UBLOX::_parse_gps(void)
         next_fix = state.status;
 #endif
         break;
+    case MSG_TIMEGPS:
```

Comment:
You need to add a call for `check_new_itow` here, as the order of messages aren't guaranteed.

Reply:
Added